### DISCUSSION TITLE
Maximum of 10 collections?

### DISCUSSION THREAD

**Discussion by leclou69** [dani-garcia/vaultwarden#4953] [Q&A] [answered]
Is it right that there is a max of 10 collections per organization?
When I try to create an 11th, I get a message that I need to purchase the pro version.
Using docker version 2024.5.1
Have I missed a configuration option or is this true?
Thanks in advance

**Comment by stefan0xC** [comment, marked as answer]
Please update your vaultwarden version to the latest one this has been fixed already.
Cf. https://github.com/dani-garcia/vaultwarden/issues/1180

**Comment by BlackDex** [comment]
No, update your instance to the latest version. This was a bug and is fixed already
